grafana/make_grafana_webpack.sh

```diff
@@ -0,0 +1,66 @@
+#! /bin/bash
```

Comment:
Is this file roughly the same as our update.sh but with some node specific stuff? Should we name it update.sh?

Reply:
I kept the same name as in the packaging done by the grafana team at RH.